public/app/plugins/datasource/prometheus/querybuilder/components/MetricSelect.tsx

```diff
@@ -131,6 +133,7 @@ export function MetricSelect({
       return (
         <div
           {...props.innerProps}
+          ref={props.innerRef}
```

Comment:
I just wonder why we need this. Could you explain it to me please?

Reply:
The innerRef and innerProps properties carry forward things like the hover and onClick needed by the option. Without them I would have to add custom styles to match the behavior of the select.